Formula/torchvision.rb

```diff
@@ -59,9 +59,9 @@ def install
       %w[-fopenmp]
     end
     system ENV.cxx, "-std=c++14", "test.cpp", "-o", "test", *openmp_flags,
-                    "-I#{libtorch.opt_include}",
-                    "-I#{libtorch.opt_include}/torch/csrc/api/include",
-                    "-L#{libtorch.opt_lib}", "-ltorch", "-ltorch_cpu", "-lc10",
+                    "-I#{pytorch.opt_include}",
+                    "-I#{pytorch.opt_include}/torch/csrc/api/include",
+                    "-L#{pytorch.opt_lib}", "-ltorch", "-ltorch_cpu", "-lc10",
```

Comment:
I think on Linux you'll also need to add `-Wl,-rpath,#{pytorch.opt_lib}` for this test to work.

Reply:
The test is working with the existing `libtorch` formula, so I think it's fine